brave/brave-core
# Migrated colors that used in theme_properties to color provider pipeline

https://github.com/brave/brave-browser/issues/23423

Introduced for color mixers for light/dark/private/tor window for using
our theme colors from color provider.

With this PR, brave's background color of new tab button/tab search button
is used.


Resolves 

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

`npm run test brave_browser_tests --filter=BraveThemeServiceTest.ColorProviderTest`

1. Launch browser and check the background of new tab button in all windows(light, dark, private, private tor)
2. Install any external theme extension from web store and check proper new tab button color is applied
